samtranslator/swagger/swagger.py

```diff
@@ -691,10 +696,15 @@ def add_auth_to_method(self, path: str, method_name: str, auth: Dict[str, Any],
         method_scopes = auth and auth.get("AuthorizationScopes")
         api_auth = api and api.get("Auth")
         authorizers = api_auth and api_auth.get("Authorizers")
+        required_options_api_key = method_name == "options" and (auth.get("AddApiKeyRequiredToCorsPreflight") is False)
```

Comment:
To improve legibility, code should be closest to where it's used. So unless it's needed elsewhere having this right above the `if` is preferred.